lightning/src/util/config.rs

```diff
@@ -126,6 +126,29 @@ pub struct ChannelHandshakeConfig {
 	///
 	/// [`KeysInterface::get_shutdown_scriptpubkey`]: crate::chain::keysinterface::KeysInterface::get_shutdown_scriptpubkey
 	pub commit_upfront_shutdown_pubkey: bool,
+
+	/// `their_channel_reserve_proportion_millionth` is proportion of channel_value
```

Comment:
Nit: 
```suggestion
	/// `their_channel_reserve_proportion_millionth` is the proportion of the channel_value
```

Pardon me if the current text is correct grammar already, as english isn't my native language. Feel free to ignore this nit if thats the case :).

Reply:
i think "`their_channel_reserve_proportion_millionth` is proportion of the channel_value" makes sense acc. to casual english.